examples/reference/widgets/CheckButtonGroup.ipynb

```diff
@@ -16,26 +16,28 @@
    "cell_type": "markdown",
```

Comment:
There is no `label`

![image](https://github.com/user-attachments/assets/d4c961aa-b720-41d0-b5bb-24a88484b0fc)